src/Microsoft.DotNet.Wpf/src/ReachFramework/PrintConfig/PTProvider.cs

```diff
@@ -238,12 +238,8 @@ public override MemoryStream GetPrintCapabilities(MemoryStream printTicket)
 
                     if (hResult == (uint)NativeErrorCode.E_PRINTTICKET_FORMAT)
                     {
-                        throw new ArgumentException(String.Format(CultureInfo.CurrentCulture,
-                                      "{0} {1} {2}",
-                                      PrintSchemaTags.Framework.PrintTicketRoot,
-                                      PTUtility.GetTextFromResource("FormatException.XMLNotWellFormed"),
-                                      errorMsg),
-                                      "printTicket");
+                        throw new ArgumentException($"{PrintSchemaTags.Framework.PrintTicketRoot} {PTUtility.GetTextFromResource("FormatException.XMLNotWellFormed")} {errorMsg}",
```

Comment:
I prefer the previous way here. The current change results in too long interpolation. Same suggestion for the change below and `FallbackPTProvider.cs`